bootstrap/helpers/databases.php

```
@@ -214,7 +214,9 @@ function deleteBackupsS3(string|array|null $filenames, S3Storage $s3): void
         'aws_url' => $s3->awsUrl(),
     ]);
 
-    $disk->delete($filenames);
+    if (! $disk->delete($filenames)) {
+        throw new RuntimeException('One or more S3 backup files could not be deleted.');
```

PHPMD (2.15.0)

[error] Missing class import via use statement (line '218', column '19'). (undefined)

(MissingImport)